Mathlib/Analysis/Calculus/ContDiff/Bounds.lean

```diff
@@ -120,6 +120,7 @@ theorem ContinuousLinearMap.norm_iteratedFDerivWithin_le_of_bilinear_aux {Du Eu
       (fun i j => ‖iteratedFDerivWithin 𝕜 i f s x‖ * ‖iteratedFDerivWithin 𝕜 j g s x‖) n).symm
 #align continuous_linear_map.norm_iterated_fderiv_within_le_of_bilinear_aux ContinuousLinearMap.norm_iteratedFDerivWithin_le_of_bilinear_aux
 
+set_option maxHeartbeats 400000 in
```

Comment:
this proof was at 206278 heartbeats before, and went to 230612; I wonder if `Qq` is slow here :/ or maybe just trying to check for positivity is too much. I wonder if it's worth optimising `positivity` so you can also "request" specific things from extensions; in this case it would save a lot of work to not check for `0 < _` unless needed.

Reply:
I also wonder if it'll get better after the fun_like changes.